app/golem.py

```diff
@@ -22,6 +22,12 @@ def node_name(self):
 
     def version(self):
         return self._version["current"]["version"]
+
+    def update(self):
+        if (self._version["pending"] == None):
+            return False

```

Comment:
Instead of returning false, let's return null since as on the receiver end it's easier to delineate between null and a string than a string and false.

Reply:
lgtm. I've updated to `return`